controllers/managedocs_controller_test.go

```diff
@@ -434,11 +434,96 @@ var _ = Describe("ManagedOCS controller", func() {
 				Expect(configMap.Data["test-key"]).Should(Equal("test-value"))
 			})
 		})
+		When("there is no enable-mcg field in the add-on parameters secret", func() {
+			It("should not create reconciled resources", func() {
```

Comment:
As you only check for storagecluster, it is best to explicitly specify it:
```suggestion
			It("should not create storagecluster", func() {
```